### PR TITLE
RDM-6380 : Implement functional test cases for "Fetch an event trigger in the context of a case type" endpoint

### PR DESCRIPTION
**Pull request by aashishaggarwal123**
# JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-6380

Change description
Implement functional test cases for "Fetch an event trigger in the context of a case type" operation until it has at least a 'good' level of functional test coverage.

Does this PR introduce a breaking change? (check one with "x")

[ ] Yes
[x] No

### CODE REVIEW

File: src/aat/resources/features/F-029/F-029.feature [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+   @S-551
+   Scenario: must return appropriate negative response when Jurisdiction does not exist
+     Given a user with [an active profile in CCD]
+     When a request is prepared with appropriate values
+     And the request [contains a non-existing Jurisdiction id]
+     And it is submitted to call the [Fetch an event trigger in the context of a case type for Case Worker] operation of [CCD Data Store]
+     Then a negative response is received
+     And the response [has the 403 return code]
+     And the response has all the details as expected
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-029/F-029.feature]: EOL at end of file

File: src/aat/resources/features/F-029/S-050.td.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "title": "must return appropriate negative response when request does not provide a valid authentication credentials",
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-029/S-050.td.json]: little grammar tweak

File: src/aat/resources/features/F-029/S-051.td.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "title": "must return appropriate negative response when request does not provide an authorized access",
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-029/S-051.td.json]: better wording please - it's not a request to provide an authorised or unauthorised access.

File: src/aat/resources/features/F-029/S-053.td.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  "_extends_": "F-029_Test_Data_Base",
+
+  "specs": [
+    "an active profile in CCD",
+    "contains a non-existing event-id",
+    "has the 404 return code"
+  ],
+  "request": {
+    "pathVariables": {
+      "etid": "CREATE123"
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-029/S-053.td.json]: maybe you may use a more arbitrary value

File: src/aat/resources/features/F-029/F-029.feature [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    And it is submitted to call the [Fetch an event trigger in the context of a case type for Case Worker] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [has the 200 return code]
+    And the response has all the details as expected
+
+  @S-050
+  Scenario: must return appropriate negative response when request does not provide a valid authentication credentials
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [does not provide a valid authentication credentials]
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-029/F-029.feature]: does not provide valid authentication credentials]

File: src/aat/resources/features/F-029/F-029.feature [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+  @S-049
+  Scenario: must return successfully a CaseEventTrigger object filtered by case type ID
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [is prepared with valid input parameters]
+    And it is submitted to call the [Fetch an event trigger in the context of a case type for Case Worker] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [has the 200 return code]
+    And the response has all the details as expected
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-029/F-029.feature]: all other details

File: src/aat/resources/features/F-029/F-029.feature (hunk 1 above)
Comment (aashishaggarwal123) [reply src/aat/resources/features/F-029/F-029.feature]: updated

File: src/aat/resources/features/F-029/S-050.td.json (hunk 2 above)
Comment (aashishaggarwal123) [reply src/aat/resources/features/F-029/S-050.td.json]: updated

File: src/aat/resources/features/F-029/S-051.td.json (hunk 3 above)
Comment (aashishaggarwal123) [reply src/aat/resources/features/F-029/S-051.td.json]: updated.

File: src/aat/resources/features/F-029/S-053.td.json (hunk 4 above)
Comment (aashishaggarwal123) [reply src/aat/resources/features/F-029/S-053.td.json]: updated

File: src/aat/resources/features/F-029/F-029.feature (hunk 5 above)
Comment (aashishaggarwal123) [reply src/aat/resources/features/F-029/F-029.feature]: updated

File: src/aat/resources/features/F-029/F-029.feature (hunk 6 above)
Comment (aashishaggarwal123) [reply src/aat/resources/features/F-029/F-029.feature]: updated

**Pull request merged**